RHACS Installation - part 1

gaurav-nelson opened the pull request:
Includes new:
- assembly `installing/index.adoc`.
- module `modules/acs-requirements.adoc`
- module `modules/add-helm-repository.adoc`
- module `modules/central-requirements.adoc`

I've broken down the installation guide into pieces for easier review. Once we merge this one, I'll add more to it.

gaurav-nelson commented:
@bergerhoffer PTAL

bergerhoffer reviewed (commented):
Really nice job breaking it up and everything. Just a few minor things, mostly style-related things.

Once you address whatever you can, just let me know if you're ready for it to be merged if it looks good. Just remember to squash to a single commit as well. Thanks!

gaurav-nelson commented:
@bergerhoffer I've addressed your comments. In addition to those, here are the changes from my latest push:
- Removed all files from the `_unused_topics` directory. And that's why the size/XXL label.
- Moved prerequisites in a separate page. Because, there are more prerequisites for each of the components (I'll add those in another PR).
- I've created an `:ocp-supported-version: 4.5` variable (Thanks). For now it list `4.5`, but I'll check and replace it with the correct version if there are changes.

gaurav-nelson commented on modules/acs-quick-install-using-helm.adoc:
```diff
@@ -0,0 +1,56 @@
+The sample applications listed in the following section contains critical vulnerabilities and they are specifically designed to verify the build and deploy-time assessment features of {product-title}.
+====
+
+.Verification on OpenShift
```
```suggestion
.Verification on {ocp}
```

gaurav-nelson commented on modules/acs-requirements.adoc:
```diff
@@ -0,0 +1,38 @@
+However, you can use another storage type if you do not have SSDs available.
+* Helm command-line interface (CLI) v3.2 or newer.
+Use the `helm version` command to verify the version of Helm you have installed.
+* The OpenShift CLI (`oc`).
```
```suggestion
* The {ocp} CLI (`oc`).
```

bergerhoffer replied:
We actually do just call it the "OpenShift CLI" (and not the "OpenShift Container Platform CLI").

bergerhoffer commented on modules/acs-quick-install-using-helm.adoc:
```diff
@@ -0,0 +1,56 @@
+:_module-type: PROCEDURE
+[id="installing-quickly_{context}"]
+= Installing quickly
+
```
Try to add an intro sentence here if you can.

bergerhoffer commented on installing/installing-using-helm.adoc:
```diff
@@ -0,0 +1,27 @@
+
+toc::[]
+
+{product-title} installs as a set services on your {ocp} cluster.
```
Should this be " a set of services"?

bergerhoffer commented on installing/installing-using-helm.adoc:
```diff
@@ -0,0 +1,27 @@
+. Configure the `secured-cluster-services` Helm chart.
+. Install xref:../architecture/index.adoc#per-cluster-components_acs-architecture[per-cluster] and xref:../architecture/index.adoc#per-node-components_acs-architecture[per-node] components (Sensor, Admission Controller, and Collector).
+
+Before you install:
```
Consider making this a "== Prerequisites" heading instead.

For example, like: https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-default.html

gaurav-nelson replied:
It's not prerequisites, it's the installation flow, so users will need to do the following tasks to install RHACS. Each task is a procedure and it will appear as a heading on this page.

bergerhoffer reviewed (commented):
Looking good, just a few more pieces of feedback.

bergerhoffer reviewed (commented):
LGTM!
